streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -111,23 +111,18 @@ public final ThreadCache cache() {
     }
 
     public abstract void commit();
-
+    public abstract void suspend();
```

Comment:
I think we should introduce an interface, `Task`? Doesn't necessarily need to be done in this PR, though